digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/NodesAndJunctionsService.scala

```diff
@@ -141,6 +141,22 @@ class NodesAndJunctionsService(roadwayDAO: RoadwayDAO, roadwayPointDAO: RoadwayP
     }
   }
 
+  /**
+    * Check if any of the junction points is on a reserved road part (VIITE-2518).
+    * @param junctionPointIds Junction point ids
```

Comment:
Noooou! Not just repeating the param name, please! :D
"Ids of the junction points to be checked."
Or sth like that.